pkg/neorpc/filters.go

```diff
@@ -49,6 +54,16 @@ type (
 	}
 )
 
+// SubscriptionFilter is an interface for all subscription filters.
+type SubscriptionFilter interface {
```

Comment:
Looks strange with the only validity check method. But maybe.

Reply:
We may extend it in future with the rest of filter-related methods, but it requires refactoring of the whole set of these methods.